apps/wallet-mobile/src/components/Modal/ModalScreen.tsx

```diff
@@ -16,20 +17,27 @@ import {useModal} from './ModalContext'
 
 export const ModalScreen = () => {
   const {current} = useCardAnimation()
-  const {height, closeModal, content} = useModal()
+  const {height, closeModal, content, isOpen} = useModal()
```

Comment:
won't require changes in all screens that consume it?

Reply:
Not really because it is just a variable to know the state of the modal.

The rest of the places where the modal is used only use the open modal / close modal actions

Reply:
Another way to do it would have been to check if the context variable content is undefined or not, but I think this is more explicit.